tests/ui/rename.rs

```diff
@@ -10,6 +10,7 @@
 #![allow(clippy::box_collection)]
 #![allow(clippy::redundant_static_lifetimes)]
 #![allow(clippy::cognitive_complexity)]
+#![allow(clippy::derived_hash_with_manual_eq)]
```

Comment:
These changes currently make the CI fail. You can automatically update the `.stderr` file by first running `cargo uitest` and den `cargo dev bless`. That should update everything